packages/system/src/Item/api/api.ts

```diff
@@ -1,10 +1,10 @@
 import {
+  ItemNodeType,
```

Comment:
I'm somewhaaat tempted to make the folder structure closer to `item/service`, `item/stock`, `item` and putting the specific things in each folder but it seemed OTT